tests/binarize_test.py

```diff
@@ -0,0 +1,49 @@
+import torch
+
+from asteroid.binarize import Binarize
+
+
+def test_Binarize():
+    inputs_list = [
+        torch.Tensor(
+            [
+                0.1,
+                0.6,
+                0.2,
+                0.6,
+                0.1,
+                0.1,
+                0.1,
+                0.7,
+                0.7,
+                0.7,
+                0.1,
+                0.7,
+                0.7,
+                0.7,
+                0.1,
+                0.8,
+                0.9,
+                0.2,
+                0.7,
+                0.1,
+                0.1,
+                0.1,
+                0.8,
+                0.1,
+            ]
```

Comment:
Black wants it not sure this is ok

Reply:
You can disable formatting by changing the last line to

```
]  # fmt: off
```